DOCS: update for lightspeed app docs update 1.1.0 (#1541)
MultiSafepay/docs: content/payments/integrations/ecommerce-platforms/lightspeed_app/faq/disabling-payment-methods.md

---

### Disabling payment methods

> You can disable payment methods in the **Payment method settings list**. Enabled payment methods show a green dot, when disabled its grey.

Reviewer: ```suggestion
You can disable payment methods in the **Payment method settings list**. Payment methods display a green dot when enabled, and are greyed out when disabled.
```
